scripts/exampleSetup.ts

```diff
@@ -36,65 +37,33 @@ async function main() {
   const gaugeHelperContract = (await ethers.getContract(
     "GaugeHelperContract",
   )) as GaugeHelperContract
-  const gauge_map = new Map<string, string>([
-    [
-      "LiquidityGaugeV5_SaddleALETHPoolLPToken",
-      (await ethers.getContract("LiquidityGaugeV5_SaddleALETHPoolLPToken"))
-        .address,
-    ],
-    [
-      "LiquidityGaugeV5_SaddleBTCPoolV2LPToken",
-      (await ethers.getContract("LiquidityGaugeV5_SaddleBTCPoolV2LPToken"))
-        .address,
-    ],
-    [
-      "LiquidityGaugeV5_SaddleD4PoolLPToken",
-      (await ethers.getContract("LiquidityGaugeV5_SaddleD4PoolLPToken"))
-        .address,
-    ],
-    [
-      "LiquidityGaugeV5_SaddleSUSDMetaPoolUpdatedLPToken",
-      (
-        await ethers.getContract(
-          "LiquidityGaugeV5_SaddleSUSDMetaPoolUpdatedLPToken",
-        )
-      ).address,
-    ],
-    [
-      "LiquidityGaugeV5_SaddleTBTCMetaPoolUpdatedLPToken",
-      (
-        await ethers.getContract(
-          "LiquidityGaugeV5_SaddleTBTCMetaPoolUpdatedLPToken",
-        )
-      ).address,
-    ],
-    [
-      "LiquidityGaugeV5_SaddleUSDPoolV2LPToken",
-      (
-        await ethers.getContract("LiquidityGaugeV5_SaddleUSDPoolV2LPToken")
-      ).address.toString(),
-    ],
-    [
-      "LiquidityGaugeV5_SaddleWCUSDMetaPoolUpdatedLPToken",
-      (
-        await ethers.getContract(
-          "LiquidityGaugeV5_SaddleWCUSDMetaPoolUpdatedLPToken",
-        )
-      ).address.toString(),
-    ],
-  ])
-  // array of all keys in gauge_addresses
-  const gauge_names = Array.from(gauge_map.keys())
-  // array of all values in gauge_addresses
-  const gauge_addresses = Array.from(gauge_map.values())
```

Comment:
Hmm this seems silly to create a map and then split them into keys and addresses right away.

I made changes to slimdown the lines by dynamically creating the "map" and keep using it instead of splitting them into key value arrays.